fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl

```diff
@@ -0,0 +1,230 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: fgbio_filter_consensus_reads_1_2_0
+baseCommand:
+  - fgbio
+inputs:
+  - id: memory_per_job
+    type: int?
+    doc: Memory per job in megabytes
+  - id: memory_overhead
+    type: int?
+    doc: Memory overhead per job in megabytes
+  - id: number_of_threads
+    type: int?
+  - id: input
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '--input'
+      shellQuote: false
+    doc: The input SAM or BAM file.
+  - id: output_file_name
+    type: string?
+    doc: Output SAM or BAM file to write consensus reads.
+  - id: reference_fasta
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '--ref'
+    doc: Reference fasta file.
+    secondaryFiles:
+      - ^.fai
+      - ^.dict
+  - id: reverse_per_base_tags
+    type: boolean?
+    inputBinding:
+      position: 0
+      prefix: '--reverse-per-base-tags'
+    doc: 'Reverse [complement] per base tags on reverse strand reads.'
+  - id: min_reads
+    type: 'int[]?'
+    inputBinding:
+      position: 0
+      prefix: '--min-reads'
+      itemSeparator: ' '
+    doc: >-
+      The minimum number of reads supporting a consensus base/read. (Max 3
+      values)
+  - id: max_read_error_rate
+    type: 'float[]?'
+    inputBinding:
+      position: 0
+      prefix: '--max-read-error-rate'
+      itemSeparator: ' '
+    doc: >-
+      The maximum raw-read error rate across the entire consensus read. (Max 3
+      values)
+  - id: max_base_error_rate
+    type: 'float[]?'
+    inputBinding:
+      position: 0
+      prefix: '--max-base-error-rate'
+      itemSeparator: ' '
+    doc: The maximum error rate for a single consensus base. (Max 3 values)
+  - id: min_base_quality
+    type: int?
+    inputBinding:
+      position: 0
+      prefix: '--min-base-quality'
+    doc: Mask (make N) consensus bases with quality less than this threshold.
+  - id: max_no_call_fraction
+    type: float?
+    inputBinding:
+      position: 0
+      prefix: '--max-no-call-fraction'
+    doc: Maximum fraction of no-calls in the read after filtering
+  - id: min_mean_base_quality
+    type: int?
+    inputBinding:
+      position: 0
+      prefix: '--min-mean-base-quality'
+    doc: The minimum mean base quality across the consensus read
+  - id: require_single_strand_agreement
+    type: boolean?
+    inputBinding:
+      position: 0
```

Comment:
I'm not sure if this is Rabix automatically adding the `position: 0` field for every input, but technically we don't need that if we have `prefix: `.